packages/x-charts/src/internals/plugins/featurePlugins/useChartCartesianAxis/useChartCartesianAxis.types.ts

```diff
@@ -47,6 +48,16 @@ export interface UseChartCartesianAxisParameters<S extends ScaleName = ScaleName
    * @param {null | ChartsAxisData} data The data about the clicked axis and items associated with it.
    */
   onAxisClick?: (event: MouseEvent, data: null | ChartsAxisData) => void;
+  /**
+   * The function called when pointer moves from one axis value to another.
```

Comment:
> when pointer moves from one axis value to another

Is this the only case? Can't it be called if the data changes as well?

I think something like "when the axis value changes" would work better. 

----

Also, we use "axis value" here, but "axis item" below. Are they the same thing? Should we use the same name in both places?

Reply:
I misunderstood your initial comment.

I was focusing on the fact I remove the `value` to only keep the `dataIndex` attribute.

> Can't it be called if the data changes as well?

I would be great but not implemented in this PR. I will have a look if the new selector implementation makes it easier to do